web/app/view/StateController.js

```diff
@@ -39,6 +43,14 @@ Ext.define('Traccar.view.StateController', {
                 },
                 '#Positions': {
                     clear: 'clearReport'
+                },
+                '#AllAttributeAliases': {
```

Comment:
At this point I thought... why do we need two stores? Maybe we can just use one which would have all aliases. And then we can filter (use chain store for it) it to display aliases per device. What do you think?